hydra-node/json-schemas/logs.yaml

```diff
@@ -1112,6 +1131,85 @@ definitions:
           event:
             $ref: "#/definitions/Event"
 
+  Outcome:
+    description: >-
+      The result of Head protocol processing Events. Each Outcome
+      represents an action to be taken by the node to transition
+      the protocol.
+    oneOf:
+      - title: OnlyEffects
+        type: object
+        additionalProperties: false
+        required:
+          - tag
+          - effects
+        description: >-
+          An outcome representing the list of actions to be taken by the node 
+          to transition the protocol.
+        properties:
+          tag:
+            type: string
+            enum: ["OnlyEffects"]
+          effects:
+            type: array
+            items:
+              type: object
+              $ref: "#/definitions/Effect"
+      - title: NewState
+        type: object
+        additionalProperties: false
+        required:
+          - tag
+          - headState
+          - effects
+        description: >-
+          An outcome representing the list of actions to be taken by the node 
+          to transition the protocol, and the new resulting state.
+        properties:
+          tag:
+            type: string
+            enum: ["NewState"]
+          headState:
+            type: object
+            $ref: "#/definitions/HeadState"
+          effects:
+            type: array
+            items:
+              type: object
+              $ref: "#/definitions/Effect"
+      - title: Wait
+        type: object
+        additionalProperties: false
+        required:
+          - tag
+          - reason
+        description: >-
+          An outcome representing the time to wait by the node before re-enqueuing
```

Comment:
It's not representing the time it waits? It's just about re-enqueuing.

Reply:
rewriting